dev-resources/vectors/kx/alice-public-key

```diff
@@ -0,0 +1 @@
+8520f0098930a754748b7ddcb43ef75a0dbf3a0d26381af4eba4a98eaa9b4e6a
```

Comment:
Did these test vectors come from kalium? If so, please add a README file to the vector folder to about where you got these from. (Ideally it'd be a shell script to download the vectors from a known-good source, but I appreciate that is not always possible :-))

Reply:
I added some C code to show how I generate the vectors, and a note about where they come from. Unfortunately it also showed that my tests were not exhaustive, or are incorrect. I'll try and rework it when I have time.